puzzle/models/sql/models.py

```diff
@@ -27,11 +27,18 @@ class Case(BASE):
     pedigree = Column(String) # For storing madeline info
     compressed=Column(Boolean, default=False)
     tabix_index=Column(Boolean, default=False)
-    
+
     created_at = Column(DateTime, default=datetime.now)
     updated_at = Column(DateTime, default=datetime.now, onupdate=datetime.now)
 
     @property
+    def case_comments(self):
+        """Return only comments made on the case."""
+        comments = (comment for comment in self.comments if
+                    comment.variant_id is None)
```

Comment:
indeed, this is how we distinguish comment types

a comment should only be able to belong to a case or a case+variant